scripts/LVGLImage.py

```diff
@@ -341,9 +341,9 @@ def from_binary(self, data: bytes):
             raise FormatError("invalid header length")
 
         try:
-            self.cf = ColorFormat(data[0] & 0x1f)  # color format
+            self.cf = ColorFormat(data[1] & 0x1f)  # color format
         except ValueError as exc:
-            raise FormatError("invalid color format") from exc
+            raise FormatError(f"invalid color format: {hex(data[0])}") from exc
```

Comment:
Shouldn't it be `data[1]` (the color format byte), that is mentioned also in PR description?

Reply:
Ops. Must be a tough day for me. Will fix it soon.